sdk/logical/version.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package logical;
+
+option go_package = "github.com/hashicorp/vault/sdk/logical";
+
+message Empty {}
+
+// VersionReply is the reply for the Version method.
+message VersionReply {
+  string plugin_name = 1;
+  string version = 2;
```

Comment:
I originally wanted to use version everywhere, so started sprinkling that around. But then we've found a couple of places where we need to use plugin_version to avoid ambiguity. I'm wondering if we should standardise on plugin_version everywhere for consistency, and for an abundance of clarity?

Reply:
I think that makes sense. I'll rename it.